src/main/scala/org/allenai/scholar/metrics/ErrorAnalysis.scala

```diff
@@ -0,0 +1,41 @@
+package org.allenai.scholar.metrics
+
+import java.time.Year
+
+import org.allenai.scholar._
+import org.allenai.scholar.metrics.PrecisionRecall._
+
+case class ErrorAnalysis(
+  metricName: String,
+  pr: PR,
+  details: Iterable[(String, Example[_])]
+)
+
+object ErrorAnalysis {
+  def computeMetrics[T](
```

Comment:
All metrics calculations are done by invoking this method.  The `metrics` function takes a data point (e.g. a `PaperMetadata`) and creates an object to be compared to the truth (e.g. a `Title` or `Author`)